osm2lanes/src/test.rs

```diff
@@ -478,6 +478,31 @@ mod tests {
         });
     }
 
+    #[test]
+    fn test_json() {
```

Comment:
This was motivated by tracking down where the serde flattening bug was happening, right?

Reply:
yeah, if you replace this with bincode, the test fails :)